pandas/core/strings.py

```diff
@@ -1950,11 +1951,14 @@ def _validate(data):
         # see _libs/lib.pyx for list of inferred types
         allowed_types = ["string", "empty", "bytes", "mixed", "mixed-integer"]
 
-        values = getattr(data, "values", data)  # Series / Index
+        values = extract_array(data)  # unpack Series / Index
```

Comment:
This is not always equivalent, eg in case of periods. But I suppose the consequent error that it gives it catched by the try/except call you added below

Reply:
I think this got included by accident,  meant for another branch.  But in general, I think extract_array should be the first-try way to replace this pattern: it's performant and non-lossy.